README.md

```diff
@@ -16,17 +16,37 @@ To guide development we [have documented the user needs](docs/NEEDS.md) and [ass
 
 ## Usage
 
+To run a GOV.UK application, `make` it first (you only need to do this once per application):
```

Comment:
This isn't necessary. If you do `govuk-docker build` in the app it will build the app + dependencies.

You can also do this from any directory:

```
govuk-docker build --service collections-publisher
```